config/routes.rb

```diff
@@ -2,6 +2,10 @@
   get "taskees/index"  => "taskees#index", as: :taskees
 
   root "pages#index"
+  get "admin/dashboard" => "dashboard#home", as: "admin_dashboard"
+  get "admin/view_applications" => "admin#view_applications", as: "applications"
+  get "admin/view_applications/:id/review_applicant" => "admin#review_applicant", as: "review_applicant"
+  post "admin/view_applications/:id" => "admin#update_applicant", as: "update_applicant"
```

Comment:
Why are we not using Rails standard RESTful routing (also using resources) here?